roles/ovirt-engine-setup/templates/answerfile_3.6_basic.txt.j2

```diff
@@ -0,0 +1,79 @@
+# action=setup
+[environment:default]
+OVESETUP_DIALOG/confirmSettings=bool:True
+OVESETUP_CONFIG/applicationMode=str:both
+OVESETUP_CONFIG/remoteEngineSetupStyle=none:None
+OVESETUP_CONFIG/sanWipeAfterDelete=bool:False
+OVESETUP_CONFIG/storageIsLocal=bool:False
```

Comment:
Do we need to have all of the options listed here? Can we just have those which can be changed and run with `accept-defaults` ? Or is there any issue with that?

Reply:
i think its there because it is needed..

Reply:
I just wonder what will happen, if there will be introduced new option in zstream, it can stop to work, so I think the accept defaults is better way, but I don't have deep knowledge of this. Just asking.

Reply:
I tend to agree with accept default for oVirt side, as the playbook accomodates to it autoamtically.
I tend to disagree with accept default for oVirt QA side, as we would need to track changes manually., as we want to know what are we testing and new options we might miss new options.

Reply:
I think you should create one common file and use `include` jinja statement, better than copy-pasting. And change only things which are relevant for the verison, this is unreadeble.

Reply:
Hmm, I think it's better to have one answer file per version as variable might be renamed between versions and no backward compatibility is ensured, so having only 1 answer could complicate things significantly.

But I think, we should provide a way how to use --accept-defaults and override only variables that user selects (for example disable OVN, but leave the rest on defaults). But this can be done in separate RFE

Reply:
I created a basic answer file and already tested on 4.1 and 4.2 and I don't see issues. I think it's good idea and definitely makes it much easier to read.

Reply:
'--accept-defaults' already behaves like that - it obeys all var/val in the answer file, and "accepts the default" only if not supplied.